src/modules/authentication/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,5 +1,212 @@
+import { useEffect, useState } from "react";
+import logo from "../../../../assets/images/PMS-logo.png";
+import { useForm } from "react-hook-form";
+import { PasswordValidation } from "../../../../validations";
+import { toast } from "react-toastify";
+import { AUTH_URLS, axiosInstance, HEADERS } from "../../../../api";
+
+import { useNavigate } from "react-router-dom";
+
 function ChangePassword() {
-	return <div></div>;
+  interface formDataInputs {
```

Comment:
We can make the type outside of the component. I also suggest having a better name for the type like `ChangePasswordPayload`